addons/mail/static/tests/chatter_tests.js

```diff
@@ -624,6 +624,81 @@ QUnit.test('chatter: post, receive and star messages', function (assert) {
         });
 });
 
+QUnit.test('chatter: post a message disable the send button', function(assert) {
+    assert.expect(3); // nombre total de test
+
+    var messages = [];
+    var bus = new Bus();
+    var getSuggestionsDef = $.Deferred();
+    var form = createView({
+        View: FormView,
+        model: 'partner',
+        data: this.data,
+        arch: '<form string="Partners">' +
+                '<sheet>' +
+                    '<field name="foo"/>' +
+                '</sheet>' +
+                '<div class="oe_chatter">' +
+                    '<field name="message_ids" widget="mail_thread" options="{\'display_log_button\': True}"/>' +
+                '</div>' +
+            '</form>',
+        res_id: 2,
+        mockRPC: function (route, args) {
+            if (args.method === 'message_get_suggested_recipients') {
+                return $.when({2: []});
+            }
+            if (args.method === 'get_mention_suggestions') {
+                getSuggestionsDef.resolve();
+                return $.when([{email: "test@odoo.com", id: 1, name: "Test User"}]);
+            }
+            return this._super(route, args);
+        },
+        session: {},
+        intercepts: {
+            get_messages: function (event) {
+                event.stopPropagation();
+                var requested_msgs = _.filter(messages, function (msg) {
+                    return _.contains(event.data.options.ids, msg.id);
+                });
+                event.data.callback($.when(requested_msgs));
+            },
+            post_message: function (event) {
+                event.stopPropagation();
+                assert.ok(form.$('.o_composer_button_send').prop("disabled"));
+                var msg_id = messages.length ? (messages[messages.length-1].id + 1) : 0;
+                messages.push({
+                    attachment_ids: [],
+                    author_id: ["42", "Me"],
+                    body: event.data.message.content,
+                    date: moment(), // now
+                    displayed_author: "Me",
+                    id: msg_id,
+                    is_note: event.data.message.subtype === 'mail.mt_note',
+                    is_starred: false,
+                    model: 'partner',
+                    res_id: 2,
+                });
+                bus.trigger('new_message', {
+                    id: msg_id,
+                    model: event.data.options.model,
+                    res_id: event.data.options.res_id,
+                });
+            },
+            get_bus: function (event) {
+                event.stopPropagation();
+                event.data.callback(bus);
+            },
+        },
+    });
+
+    form.$('.o_chatter_button_new_message').click();
+    assert.ok(!form.$('.o_composer_button_send').prop("disabled"));
+    form.$('.oe_chatter .o_composer_text_field:first()').val("My first message");
```

Comment:
I think triggering the event "input" is necessary if you want the widget to really take the value
´.trigger("input")´